src/components/MoneyRequestConfirmationList.js

```diff
@@ -158,6 +170,18 @@ class MoneyRequestConfirmationList extends Component {
         return _.map(participants, (option) => _.omit(option, 'descriptiveText'));
     }
 
+    /**
+     * Returns the personalDetails object for the requester
```

Comment:
Let's standardize on payer/payee

```suggestion
     * Returns the personalDetails object for the payee
```